Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2721,6 +2744,27 @@ bool ComputedStyleExtractor::hasProperty(CSSPropertyID propertyID)
     return propertyValue(propertyID);
 }
 
+static bool rendererHasTrimmedMargin(const RenderElement* renderer, CSSPropertyID propertyId)
```

Comment:
`trimmedMargin` is supposed to indicate that the box has a trimmed margin (not sure if that matches with the definition of layout output). Perhaps a better name would be `marginTrimMarkings` since the vocabulary that I have been using is that we "mark," bits

Reply:
I am not sure if markings makes it any clearer since the term "markings" does not really indicate a transition from "computed" to "result of layout" but again I am not good at namings. 🤷 
I wouldn't be too worried about this though at this point.